Bottom: b1e27f9759a3aff134521e60bc4ed98ec9bc3660
Top:    8320ea04dfbd0036e5af133b256f1a1039ecce3f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-30 18:45:18 +0100

Refresh of play-with-glsl-shader-for-rend

---

diff --git a/src/hid/common/hidgl.c b/src/hid/common/hidgl.c
index 0649fac..e3f30ea 100644
--- a/src/hid/common/hidgl.c
+++ b/src/hid/common/hidgl.c
@@ -866,6 +866,7 @@ hidgl_finish_render (void)
   if (!in_context)
     fprintf (stderr, "hidgl: hidgl_finish_render() - Not currently in rendering context!\n");
 
+  hidgl_finish_triangle_array (&buffer);
   hidgl_shader_activate (NULL);
   in_context = false;
 }
